website/server/models/task.js

```diff
@@ -58,6 +58,7 @@ export let TaskSchema = new Schema({
   userId: {type: String, ref: 'User', validate: [validator.isUUID, 'Invalid uuid.']}, // When not set it belongs to a challenge
 
   challenge: {
+    name: {type: String, default: 'No Category'},
```

Comment:
The "No Category" is weird here, IMO. Maybe use "None" or "Private" or something instead?